python_modules/dagster/dagster/_core/storage/db_io_manager.py

```diff
@@ -67,6 +67,10 @@ def delete_table_slice(context: OutputContext, table_slice: TableSlice) -> None:
     def get_select_statement(table_slice: TableSlice) -> str:
         ...
 
+    @staticmethod
+    def create_schema(context: OutputContext, table_slice: TableSlice) -> None:
```

Comment:
Maybe we should call this something like `ensure_schema_exists`? Because we won't always be creating it?